# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/rockplug/stonemod.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./etc/stone.d/mod_rockplug.sh.orig	2004-12-24 00:33:47.647396336 +0100
+++ ./etc/stone.d/mod_rockplug.sh	2004-12-24 00:34:24.846741176 +0100
@@ -19,8 +19,6 @@
 # file for details.
 # 
 # --- ROCK-COPYRIGHT-NOTE-END ---
-#
-# [MAIN] 20 rockplug ROCK-Plug/Drivers Configuration
 
 ######################################################################
 # DEFAULT CONFIG GENERATION
